query/math.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func isMathFunction(call *influxql.Call) bool {
 	switch call.Name {
-	case "sin", "cos", "tan":
+	case "sin", "cos", "tan", "floor", "ceiling", "truncate":
```

Comment:
Can you name these [ceil()](https://prometheus.io/docs/prometheus/latest/querying/functions/#ceil()), [floor()](https://prometheus.io/docs/prometheus/latest/querying/functions/#floor()), and [round()](https://prometheus.io/docs/prometheus/latest/querying/functions/#round())? I'm not sure why truncate is needed. I think you can get that behavior by using a cast such as `value::integer`. If you can, please implement round instead.

Reply:
I have pushed a new commit with rename `ceiling ` to `ceil` and remove `truncate ` and implemented `round `.